### PR TITLE
Add Latitude.sh Frankfurt as the new region

### PR DESCRIPTION
**Pull request by furkansahin**
## Move latitude-ai region under latitude provider

### CODE REVIEW

File: config/providers.yml [hunk 1]
```diff
@@ -17,5 +17,12 @@
       {display_name: us-east-a2, internal_name: leaseweb-wdc02, ui_name: "Virginia, US", visible: true},
       {display_name: us-south-a13, internal_name: leaseweb-dal13,  ui_name: "Dallas, US", visible: false},
     ]
+  },
+  {
+    provider_display_name: Latitude,
+    provider_internal_name: latitude,
+    locations: [
+      {display_name: eu-central-l1, internal_name: latitude-fra, ui_name: "Frankfurt, Germany", visible: false}
```
Comment (enescakir) [inline config/providers.yml:25]: We use `a` such as `us-east-a2` for Leaseweb. Do you know the reason for that (I don't know)? Should we use `a` for Latitude too? Lower `l` is little hard to differentiate from numbers such as `1`?

Comment (enescakir) [inline config/providers.yml:25]: Currently, Hetzner regions are named with countries, while others use both city and country names. I recall Umur initiating a discussion on this topic earlier. I can't remember the outcome, but I believe we should adhere to one naming convention.

Comment (furkansahin) [reply config/providers.yml:25]: There is already an active discussion in slack about this :)

Comment (furkansahin) [reply config/providers.yml:25]: this makes sense. I'll update

**Review by bsatzger — APPROVED** [review]

File: lib/option.rb [hunk 2]
```diff
@@ -32,8 +32,8 @@ module Option
   PROVIDERS.freeze
   LOCATIONS.freeze
 
-  def self.locations(only_visible: true)
-    Option::LOCATIONS.select { !only_visible || _1.visible }
+  def self.locations(only_visible: true, feature_flags: [])
+    Option::LOCATIONS.select { !only_visible || (_1.visible || feature_flags.include?("location_#{_1.name.gsub(/[-_]/, "_")}")) }
```
Comment (enescakir) [inline lib/option.rb:36]: Nit: no need to replace underscore with underscore
```suggestion
    Option::LOCATIONS.select { !only_visible || (_1.visible || feature_flags.include?("location_#{_1.name.tr("-", "_")}")) }
```

**Review by enescakir — APPROVED** [review]

**Pull request merged**